app/lint.xml

```diff
@@ -10,6 +10,9 @@
     <issue id="InvalidPackage">
         <!-- Ignore errors about BC importing javax.naming because we don't use those modules -->
         <ignore path="**/bcpkix-*.jar" />
+        <ignore path="**/org.jacoco/**" />
+        <ignore path="**/grpc-core*.jar" />
+        <ignore path="**/kotlinx-coroutines-core-jvm-1.5.2.jar" />
```

Comment:
Please include the reason for ignoring these packages (see Bouncy Castle entry above). This is partly to make sure we aren't silencing issues that we should actually be fixing, and partly to document the offending dependency (to remove the entry if the dependency is dropped in the future).